src/app/search/directives/series-list.component.css

```diff
@@ -0,0 +1,43 @@
+publish-series-card, .series {
+  display: block;
+  overflow: hidden;
+  position: relative;
+  margin-top: 20px;
+  margin-right: 20px;
+  flex-basis: 490px;
+  -ms-flex-preferred-size: 490px;
+  -webkit-flex-basis: 490px;
+  -webkit-box-flex: 2;
+  -moz-box-flex: 2;
+  -o-box-flex: 2;
+  -ms-box-flex: 2;
+  box-flex: 2;
+  -webkit-flex-grow: 2;
+  flex-grow: 2;
+  -webkit-flex-shrink: 1;
+  flex-shrink: 1;
+  height: 192px;
+  background: #fff;
+}
+
+/* attempt to fix the odd last item stretching
```

Comment:
Hmm... could you insert a couple of extra 0-height cards at the end of the array?  If one of them gets shoved to the last line, it wouldn't be noticeable.  And otherwise they'd just push the last real-item to the left.